lib/routers/profile/pil.js

```diff
@@ -69,7 +69,7 @@ router.param('pil', (req, res, next, id) => {
 
   Promise.resolve()
     .then(() => {
-      return PIL.query().findOne({ id, profileId: req.profileId });
+      return PIL.query().findOne({ id, profileId: req.profileId }).skipUndefined();
```

Comment:
This isn't scoped to an establishment. That is bad.

Reply:
oh yeah, derp